.github/workflows/ci.yml

```diff
@@ -6,18 +6,18 @@ jobs:
   build:
     name: JDK ${{ matrix.java_version }}
     runs-on: ubuntu-latest
-    strategy: 
+    strategy:
       matrix:
-        # TODO Add 9, 10, 11, 12, and 13 after Kotlin 1.3.60
-        java_version: [8]
+        java_version: [11]
```

Comment:
If possible, could we introduce a minor diff to update this after we resolve the `gradle publish task` like this pull request. https://github.com/uber/RIBs/pull/477/files

Reply:
Sure, I removed the changes related to AGP / Gradle / Java, and we can do it on another PR once this one is merged